docs/website/docs/command-reference/list-services.md

```diff
@@ -0,0 +1,52 @@
+---
+title: odo list services
+---
+
+## Description
+
+You can use `odo list services` to list all the bindable Operator backed services on the cluster.
+
+## Running the command
+
+To list bindable services in the current project/namespace:
+```shell
+odo list services
+```
+```shell
+$ odo list services
+ ✓  Listing bindable services from namespace "myproject" [82ms]
+
+ NAME                                               NAMESPACE 
+ redis-standalone/Redis.redis.redis.opstreelabs.in  myproject 
+```
+
+To list bindable services in all projects/namespaces accessible to the user:
+```shell
+odo list services -A 
+```
+```shell
+odo list services -A
+ ✓  Listing bindable services from all namespaces [182ms]
+
+ NAME                                               NAMESPACE  
+ redis-standalone/Redis.redis.redis.opstreelabs.in  myproject  
+ hello-world/RabbitmqCluster.rabbitmq.com           newproject 
+```
```

Comment:
Do we want to display the output for commands? Do you find it helpful? I ask this because I am working on making the docs consistent, and I can't decide if having this output is helpful or not. If we keep this, we'll need to ensure the output is up to date.

Reply:
I think they are helpful and would rather have the output here. But if it's a universal thing we are doing across the docs, I am OK to remove it.